kalamine/tpl/full.RC

```diff
@@ -0,0 +1,51 @@
+#include "winver.h"
+1 VERSIONINFO
+ FILEVERSION       ${rc_version}
+ PRODUCTVERSION    ${rc_version}
+ FILEFLAGSMASK 0x3fL
+ FILEFLAGS 0x0L
+FILEOS 0x40004L
+ FILETYPE VFT_DLL
+ FILESUBTYPE VFT2_DRV_KEYBOARD
```

Comment:
nit, is the one-space indentation on purpose ?

Reply:
I kept the format as is by kbdutools, we never know with MS tooling…